lib/OrdinaryDiffEqRosenbrock/src/rosenbrock_perform_step.jl

```diff
@@ -235,6 +256,23 @@ end
         end
         integrator.stats.nsolve += 1
 
+        if cache isa Rosenbrock32ConstantCache
```

Comment:
this is wrong. For Rosenbrock32 the 3rd stage happens without adaptivity

Reply:
Can you please suggest a change to make here?? I'll implement that accordingly

Reply:
Made the changes